docs/cv-upload-feature/tasks.md

```diff
@@ -0,0 +1,251 @@
+  - Close button (X) integration
+  - Store dismissal state in user preferences
+  - Reference: [Figma - Toast Message](https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/?node-id=32596%3A55601)
+
+- [ ] **Add dismissal toast message**
+  - Reuse existing Toast component from `packages/shared/src/components/notifications/`
+  - Message: "You can upload / edit your CV from the profile section"
+  - Include "Go to profile" action link
+
+### 2.4 Feed Integration
+- [ ] **Integrate banner into homepage feed** (`packages/webapp/src/components/Feed/`)
+  - Add conditional rendering based on CV upload status
+  - Position banner according to design specifications
+  - Ensure banner doesn't interfere with existing feed functionality
+
+## Phase 3: Profile Page Integration
+
+### 3.1 Profile CV Section
+- [ ] **Create CVProfileSection component** (`packages/shared/src/components/profile/CVProfileSection.tsx`)
+  - Empty state design: [Figma - Profile Empty](https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/?node-id=32596%3A54814)
+  - Uploaded state design: [Figma - Profile Uploaded](https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/?node-id=32596%3A54878)
+  - Loading state design: [Figma - Loading States](https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/?node-id=32597%3A3874)
+
+### 3.2 CV Management Features
+- [ ] **Add file replacement functionality**
+  - Upload new CV option
+  - Confirmation modal for replacement
+  - Delete/remove CV option
+
+- [ ] **Add CV file information display**
+  - Show filename and upload date
+  - File size and type information
+  - Upload timestamp with relative formatting
+
+### 3.3 Profile Page Integration
+- [ ] **Integrate CV section into profile page** (`packages/webapp/src/components/Profile/`)
+  - Add to existing profile form/layout
+  - Maintain consistency with other profile sections
+  - Reference: [Figma - Profile Section](https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/?node-id=32596%3A54641)
+
+## Phase 4: LinkedIn Import Flow
+
+### 4.1 LinkedIn Import Modal
+- [ ] **Create LinkedInImportModal component** (`packages/shared/src/components/modals/LinkedInImportModal.tsx`)
+  - Step-by-step guidance design: [Figma - LinkedIn Import](https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/?node-id=32596%3A54795)
+  - External link to LinkedIn profile
+  - Instructions for PDF export process
+
+### 4.2 Import Flow Integration  
+- [ ] **Add "Import from LinkedIn" trigger**
+  - Link in upload banners and profile sections
+  - "Feeling lazy?" helper text integration
+  - Modal launch functionality
+
+- [ ] **Add mobile-specific import flow**
+  - Reference: [Figma - Mobile Import](https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/?node-id=32596%3A56365)
+  - Touch-optimized interaction
+
+## Phase 5: Success & Error States
+
+### 5.1 Success Celebrations
+- [ ] **Create CVUploadSuccessModal component** (`packages/shared/src/components/modals/CVUploadSuccessModal.tsx`)
```

Comment:
Looks like it will use a lot of Modals, but not 100% sure if it knows how to use LazyModal setup. Let's see.

Reply:
I'll make sure it uses the lazy modal